src/main.rs

```diff
@@ -17,35 +19,55 @@ struct HttpServerConfig {
 
 #[tokio::main]
 async fn main() {
+    logger::init_logger();
     let configuration_wrapper: configuration_wrapper::ConfigurationWrapper =
         configuration_wrapper::ConfigurationWrapper::new().unwrap(); // unwrap should panic if the config fails to load
+    info!("Configuration File Parsed");
     let options: Result<options_parser::Options, options_parser::OptionsError> =
         options_parser::get_options();
-
+    info!("Command-line Arguments Parsed");
     let http_server_config: HttpServerConfig = configuration_wrapper
         .bind::<HttpServerConfig>("http_server")
         .unwrap_or_default();
     let server = http_server::HttpServer::new(http_server_config.host, http_server_config.port);
+    info!("HTTP Server Created");
+
+    debug!(
+        "tidyhub_address = {}",
+        configuration_wrapper
+            .bind::<String>("tidyhub_address")
+            .unwrap_or_default()
+    );
+    debug!(
+        "tidyhub_port = {}",
+        configuration_wrapper
+            .bind::<String>("tidyhub_port")
+            .unwrap_or_default()
+    );
```

Comment:
```suggestion
```

This is probably a residue from another PR, we shouldn't need that (at least here)

Reply:
This is what I expect a logger to do; to debug. Even if the main function gets bigger

Reply:
@majent4 you're right, but let's at least tidy it up a notch and put everything into one debug statement as discussed here: [https://github.com/TidyBee/tidybee-backend/pull/63#discussion_r1335005254](https://github.com/TidyBee/tidybee-backend/pull/63#discussion_r1335005254)